packages/gp2-components/src/organisms/OpenQuestionsModal.tsx

```diff
@@ -0,0 +1,137 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { Button, pixels, TextArea } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { ComponentProps, useState } from 'react';
+import { addIcon, binIcon } from '../icons';
+import { mobileQuery } from '../layout';
+import EditUserModal from './EditUserModal';
+
+const { rem } = pixels;
+
+const containerStyles = css({
+  [mobileQuery]: {
+    display: 'unset',
+  },
+  display: 'flex',
+  paddingBottom: rem(8),
+  flexDirection: 'column',
+});
+
+const rowStyles = css({
+  marginBottom: rem(16),
+  padding: `${rem(16)} 0`,
+  ':last-child': {
+    borderBottom: 'none',
+    marginBottom: 0,
+    paddingBottom: 0,
+  },
+});
+
+const headerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+});
+
+const buttonStyles = css({ margin: 0 });
+
+const addButtonStyles = css({
+  width: 'fit-content',
+  [mobileQuery]: {
+    width: '100%',
+  },
+});
+
+const optional = '(optional)';
+
+type OpenQuestionsModalProps = Pick<gp2Model.UserResponse, 'questions'> &
+  Pick<ComponentProps<typeof EditUserModal>, 'backHref'> & {
+    onSave: (userData: gp2Model.UserPatchRequest) => Promise<void>;
+  };
+
+const OpenQuestionsModal: React.FC<OpenQuestionsModalProps> = ({
+  questions,
+  backHref,
+  onSave,
+}) => {
+  const [newQuestions, setQuestions] = useState(
+    !questions.length ? [''] : questions,
+  );
+  const checkDirty = () =>
+    newQuestions.some((question, index) => question !== questions[index]);
+
+  const onChangeValue = (index: number) => (value: string) =>
+    setQuestions(
+      Object.assign([], newQuestions, {
+        [index]: value,
+      }),
+    );
+  const onRemove = (index: number) => () =>
+    setQuestions(newQuestions.filter((_, idx) => idx !== index));
+  const onAdd = () => setQuestions([...newQuestions, '']);
+
+  return (
+    <EditUserModal
+      title="Open Questions"
+      description="Share the research questions that interest you and drive your work. Other members will be able to respond to you privately using your email."
+      onSave={() => onSave({ questions: newQuestions })}
+      backHref={backHref}
+      dirty={checkDirty()}
+    >
+      {({ isSaving }) => (
+        <>
+          <div css={[containerStyles]}>
```

Comment:
I can see some css props with array and others without.
Let's make them consistent. Probably all without the array as there's only one item.